packages/twenty-front/src/modules/workflow/workflow-actions/components/WorkflowEditActionFormSendEmail.tsx

```diff
@@ -79,14 +76,13 @@ export const WorkflowEditActionFormSendEmail = ({
   };
 
   useEffect(() => {
-    form.setValue(
-      'connectedAccountId',
-      action.settings.input.connectedAccountId ?? '',
-    );
-    form.setValue('email', action.settings.input.email ?? '');
-    form.setValue('subject', action.settings.input.subject ?? '');
-    form.setValue('body', action.settings.input.body ?? '');
-  }, [action.settings, form]);
+    setFormData({
+      connectedAccountId: action.settings.input.connectedAccountId,
+      email: action.settings.input.email,
+      subject: action.settings.input.subject ?? '',
+      body: action.settings.input.body ?? '',
+    });
+  }, [action.settings.input]);
```

Comment:
After reviewing this, I would remove the useEffect. We built the form field inputs with a `defaultValue` in mind, not a `controlled` `value`. Technically, our abstractions don't allow us to set the value of TipTap editors.

This `useEffect` has no effect, at least for form fields. The connected account select is a controlled component, and this `useEffect` would set its value. I think it's better to consider that the back-end state doesn't override the local state. We'll see how we can handle it properly if that's okay with you. Preventing the local override makes the experience on a slow network better. I prefer not to override what people might have typed in inputs to display the fresher data coming from the backend.